plugins/scaffolder-backend/sample-templates/docs-template/{{cookiecutter.component_id}}/catalog-info.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   description: {{cookiecutter.description | jsonify}}
   annotations:
     github.com/project-slug: {{cookiecutter.storePath | jsonify}}
```

Comment:
This needs to be updated as well.

Reply:
To fix this, we can leverage Jinja2 for a quick hack, until we fix it properly

```jinja2
{% set listSplit = storePath.split('/') %}
{% set projectSlug = listSplit[-2] + "/" + listSplit[-1] %}

{{ projectSlug }}
```

Tested on a live parser https://cryptic-cliffs-32040.herokuapp.com